### DISCUSSION TITLE
Setup issue Invalid module "next-video\process"

### DISCUSSION THREAD

**Discussion by pat-double-u** [muxinc/next-video#173] [Q&A]
Helo, 
I tried to setup next-video on my new next.js project. I've created a new project, installed & init the next-video package. But I receive this error when I try to restart the next dev server. I don't know how to fix this. Can you help?

TypeError [ERR_INVALID_MODULE_SPECIFIER]: Invalid module "next-video\process" is not a valid package name imported from C:\...\NextJS\yadea-app\next.config.mjs
    at parsePackageName (node:internal/modules/esm/resolve:772:11)
    at packageResolve (node:internal/modules/esm/resolve:795:5)
    at moduleResolve (node:internal/modules/esm/resolve:901:20)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_INVALID_MODULE_SPECIFIER'
}

Node.js v20.10.0

next.config.mjs:

import { withNextVideo } from "next-video\\process";
/** @type {import('next').NextConfig} */
const nextConfig = {};

export default withNextVideo(nextConfig);

**Comment by luwes** [comment]
that backslash is not right I think...

could you try with the import 

`import { withNextVideo } from "next-video/process";`

or 

`import { withNextVideo } from "next-video/dist/process.js";`

**Reply by pat-double-u** [reply]
it works now, both expressions, but it throws new stuff for both...don't know to ignore or not.

 ✓ Compiled / in 7.9s (524 modules)
 ⚠ Found a change in next.config.mjs. Restarting the server to apply the changes...
   ▲ Next.js 14.1.0
   - Local:        http://localhost:3000

 ✓ Ready in 8.5s
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of C:\...\NextJS\yadea-app\node_modules\next-video\dist\config.js for build dependencies failed at 'import(fileUrl        
<w>   )'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.

**Reply by luwes** [reply]
I think it's okay to ignore but I'll look into this error.

which OS are you on and which version? and which Node version?

**Reply by pat-double-u** [reply]
allright. Thank you!
I use 
- Windows 11 Pro
- Node.js v20.10.0
- Next.js 14.1.0

I just followed the instruction on next-video readme. No adjustments
The Next project is with Typescript and ESLint.

**Reply by luwes** [reply]
following up in issue https://github.com/muxinc/next-video/issues/176
